Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -307,6 +308,98 @@ public void Generate_InvalidSqaleFileSpecified_GeneratorError()
             logger.AssertSingleErrorExists("invalidSqale.xml"); // expecting an error containing the invalid sqale file name
         }
 
+        [TestMethod]
+        public void CreatePluginManifest_TitleMissing()
```

Comment:
There should be a test for "all properties available" too (otherwise how do we know that the currently untested properties are being set?)